android/titanium/build.gradle

```
@@ -82,50 +87,45 @@ android {
 		main {
 			manifest.srcFile 'AndroidManifest.xml'
 			assets.srcDirs = [
-				'assets',
-				"${projectDir}/../modules/ui/assets"
+					'assets',
+					"${projectDir}/../modules/ui/assets"
```

Intentional indention?

```suggestion
				'assets',
				"${projectDir}/../modules/ui/assets"
```